config/samples/dspipelines.io_v1alpha1_dspipeline.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: dspipelines.opendatahub.io/v1alpha1
+kind: DSPipeline
+metadata:
+  name: sample
+spec:
+  apiServer:
+    # optional (default is: ds-pipeline-artifact-script-${metadata.name} / artifact_script)
+    artifactScriptConfigMap:
+      name: ds-pipeline-artifact-script-sample
+      key: "artifact_script"
+    apiServerImage: quay.io/modh/odh-ml-pipelines-api-server-container:v1.18.0-8
```

Comment:
Are there upstream images we can use instead?

Reply:
this should get templated, as per the jira task of adding more options to the CR spec, I think a github issue would be better for this

Reply:
sounds good.